__tests__/app.test.js

```diff
@@ -400,4 +400,18 @@ describe("app", () => {
         });
     });
   });
+  describe("DELETE /api/comments/:comment_id", () => {
+    test("status:204, no response", () => {
+      return request(app).delete("/api/comments/2").expect(204);
+    });
```

Comment:
Add a then block to this request, and inside it write a SELECT db query to try and get comment 2 and expect the response to be an empty array to make sure the deletion has been successful :)